### PR TITLE
Update http_fundamentals.rst

### PR DESCRIPTION
**Pull request by dominik-najberg**
Added "Live HTTP Headers" Chrome extension, to make it easier to find it for the Chrome browser users.

### CODE REVIEW

File: book/http_fundamentals.rst [hunk 1]
```diff
@@ -173,7 +173,7 @@ Symfony is architected to match this reality.
     To learn more about the HTTP specification, read the original `HTTP 1.1 RFC`_
     or the `HTTP Bis`_, which is an active effort to clarify the original
     specification. A great tool to check both the request and response headers
```
Comment (xabbuh) [inline book/http_fundamentals.rst]: We should now reword the sentence a bit: "Great tools to check [...] while browsing are the [...]"

File: book/http_fundamentals.rst [hunk 2]
```diff
@@ -173,7 +173,7 @@ Symfony is architected to match this reality.
...
-    while browsing is the `Live HTTP Headers`_ extension for Firefox.
+    while browsing is the `Live HTTP Headers`_ extension for Firefox or `Live HTTP Headers (Chrome)`_ extension for Chrome.
```
Comment (xabbuh) [inline book/http_fundamentals.rst]: Can you please wrap lines after the first word that crosses the 72nd character?

**Comment by javiereguiluz** [comment]
@potworny thanks for fixing this issue. As you may know, we recently revamped Symfony Docs, so your pull request can no longer be cleanly merged. I've created #6830 to fix your work, so we can close this PR.

In any case, the original tip was removed as part of the new docs ... so maybe they decide to not add it back. Let's wait and see. Thanks.

**Comment by xabbuh** [comment]
Thank you @potworny for the start.

**Comment by potworny** [comment]
No problem, mate :)

**Pull request closed without merging**